src/util/color/predefinedcolorpalettes.cpp

```diff
@@ -182,6 +203,31 @@ const ColorPalette PredefinedColorPalettes::kSeratoDJProHotcueColorPalette =
                 },
                 QList<int>{0, 2, 12, 3, 6, 15, 9, 14});
 
+const ColorPalette PredefinedColorPalettes::kMixxxTrackColorPalette =
+        ColorPalette(
+                QStringLiteral("Mixxx Track Colors"),
+                QList<mixxx::RgbColor>{
```

Comment:
Is `QList<mixxx::RgbColor>` actually needed here? I don't need to know the actual type here, the initializer list is fine.